backend/src/serverless/integrations/iterators/twitterIterator.ts

```diff
@@ -322,7 +323,7 @@ export default class TwitterIterator extends BaseIterator {
         timestamp: moment(Date.parse(record.createdAt)).utc().toDate(),
         crowdInfo: {
           sourceId: record.id,
-          body: record.text ? record.text : '',
+          body: record.text ? sanitizeHtml(record.text) : '',
```

Comment:
We should have sanitization in the `ActivityRepository.create`, and `ActivityRepository.update` as well. In the new API I already have this: https://github.com/CrowdDotDev/crowd.dev/blob/feature/new-api-activities/backend/src/database/repositories/activityRepository.ts#L22

How it is now, someone could add activities through the API with malicious HTML, having the same data structure as any of these, and then the front-end would be rendering that HTML. 

I can also add this if you need.

Reply:
Will add np.